### ISSUE TITLE
Research Paper

### ISSUE THREAD

**Issue by ucyo** [SilkeDH/lossy-ml#2]
# Table of Contents

1.  [Links](#orgff7557b)
2.  [Coding](#orgdda7a4c)
3.  [Meta compression](#org33c211e)
4.  [ML/AI for compression (or weather forecasts)](#orgf8ef8f6)
5.  [General compression](#org182cd35)
6.  [Predictive model training](#org74ecb8c)
7.  [Variable Precision](#org5c4a25e)
8.  [General](#orgf99d299)

List of research documents accumulated since the PhD.

<a id="orgff7557b"></a>

# Links

-   [#Learned Image Compression](20200407154510-learned_image_compression.md)

<a id="orgdda7a4c"></a>

# Coding

-   [Coding ANS](http://www.cs.ox.ac.uk/people/jeremy.gibbons/publications/asymm.pdf)
-   [Pulsewidth Modulation-Based Algorithm for Spike Phase Encoding and Decoding of Time-Dependent Analog Data](https://ieeexplore.ieee.org/abstract/document/8901148)
-   [SGD momentum optimizer with step estimation by online parabola model](https://www.researchgate.net/publication/334507221_SGD_momentum_optimizer_with_step_estimation_by_online_parabola_model)
-   [Efficient Bernoulli Probability Distribution Estimation for Arithmetic Coding](http://programme.meconet.me/documents/PAPERS/MECO_2020_paper_174.pdf)
-   [Variable-Radix Coding of the Reals](https://www.researchgate.net/publication/341452188_Variable-Radix_Coding_of_the_Reals?utm_medium=email&utm_source=researchgate&utm_campaign=re437&loginT=FrreQZ052pyKcE_UeEEGZHxJ0i0yjy4fVPY0uWRwLylzC_F49P8WUhpUgXVZ2Wx_HdJff3B4TPD7Min2&pli=1&utm_term=re437_x_ac&utm_content=re437_x_ac_p4&cp=re437_x_ac_p4&uid=z11QLQKL7uOkfI1TvouFgbcSo2KyeJ7X00fT&ch=reg)

<a id="org33c211e"></a>

# Meta compression

-   [Compression Ratio Modeling and Estimation across Error Bounds for Lossy Compression](https://ieeexplore.ieee.org/abstract/document/8821342)
-   [Compression Challenges in Large Scale Partial Differential Equation Solvers](http://scholar.google.de/scholar_url?url=https://www.mdpi.com/1999-4893/12/9/197/pdf&hl=en&sa=X&d=5755393719058927099&scisig=AAGBfm2KxuAsqu6AHcuNqM2OLMr5CyiO1Q&nossl=1&oi=scholaralrt)
-   [Evaluation of lossless and lossy algorithms for the compression of scientific datasets in netCDF-4 or HDF5 files](https://gmd.copernicus.org/articles/12/4099/2019/gmd-12-4099-2019.html)
-   [Z-checker (0.1.3) Compression Assessment Report for QMCPack](https://www.mcs.anl.gov/~shdi/download/z-checker-report-QMCPack-preconditioned.pdf)
-   [Z-checker (0.1.3) Compression Assessment Report for HACC](https://www.mcs.anl.gov/~shdi/download/z-checker-report-hacc-positions-abs.pdf)
-   [Z-checker (0.1.3) Compression Assessment Report for NYX data](https://www.mcs.anl.gov/~shdi/download/z-checker-report-NYX-log.pdf)
-   [Z-checker (0.1.3) Compression Assessment Report for Hurricane](https://www.mcs.anl.gov/~shdi/download/z-checker-report-Hurricane-ISABEL.pdf)
-   [Z-checker (0.1.3) Compression Assessment Report for EXAFEL](https://www.mcs.anl.gov/~shdi/download/z-checker-report-exafel.pdf)
-   [Analyzing the Impact of Lossy Compressor Variability on Checkpointing Scientific Simulations](https://ieeexplore.ieee.org/abstract/document/8891052)
-   [Evaluating Lossy Compressors for Inline Compression](https://sc19.supercomputing.org/proceedings/src_poster/poster_files/spostu106s2-file2.pdf)
-   [Analyzing the Performance of ZFP Compressed Arrays on HPC Kernels](https://sc19.supercomputing.org/proceedings/src_poster/poster_files/spostu109s2-file2.pdf)
-   [Understanding Performance-Quality Trade-offs in Scientific Visualization Workflows with Lossy Compression](https://sc19.supercomputing.org/proceedings/workshops/workshop_files/ws_drbsd103s1-file1.pdf)
-   [Estimating Lossy Compressibility of Scientific Data Using Deep Neural Networks](https://ieeexplore.ieee.org/abstract/document/8985172)
-   [Stability Analysis of Inline ZFP Compression for Floating-Point Data in Iterative Methods](https://arxiv.org/pdf/2003.02324.pdf)
-   [Exploring the feasibility of lossy compression for PDE simulations](https://experts.illinois.edu/en/publications/exploring-the-feasibility-of-lossy-compression-for-pde-simulation)

<a id="orgf8ef8f6"></a>

# ML/AI for compression (or weather forecasts)

-   [Wavefield compression for seismic imaging via convolutional neural networks](https://library.seg.org/doi/abs/10.1190/segam2019-3216395.1)
-   [Progress and Challenges for the Use of Deep Learning to Improve Weather Forecasts](https://insidehpc.com/2019/11/progress-and-challenges-for-the-use-of-deep-learning-to-improve-weather-forecasts/)
-   [Improving Data Compression with Deep Predictive Neural Network for Time Evolutional Data](https://sc19.supercomputing.org/proceedings/tech_poster/poster_files/rpost181s2-file2.pdf)
-   [Adaptive Deep Learning based Time-Varying Volume Compression](https://ieeexplore.ieee.org/abstract/document/9006146)
-   [Compression of climate simulations with a nonstationary global SpatioTemporal SPDE model](https://projecteuclid.org/euclid.aoas/1593449315)

<a id="org182cd35"></a>

# General compression

-   [Combining Checkpointing and Data Compression to Accelerate Adjoint-Based Optimization Problems](https://link.springer.com/chapter/10.1007/978-3-030-29400-7_7)
-   [Improving Performance of Data Dumping withLossy Compression for Scientific Simulation](http://tao.cs.ua.edu/paper/CLUSTER19-IOAwareLossy.pdf)
-   [A Violently Tornadic Supercell Thunderstorm Simulation Spanning a Quarter-Trillion Grid Volumes: Computational Challenges, I/O Framework, and Visualizations of Tornadogenesis](https://www.mdpi.com/2073-4433/10/10/578/htm)
-   [An information-theoretic approach toextracting climate signals from deep polarice cores](https://aip.scitation.org/doi/pdf/10.1063/1.5127211?class=pdf&)
-   [Accelerating Lossy Compression on HPC datasetsvia Partitioning Computation for Parallel Processing](http://zouxy.info/pubs/ParaSZ-HPCC-19.pdf)
-   [Lossy compression of multivariate floating-pointtime series data via improved prediction](https://arxiv.org/pdf/1911.00208.pdf)
-   [Temporal Lossy In-Situ Compression for Computational Fluid Dynamics Simulations](https://tubaf.qucosa.de/landing-page/?tx_dlf[id]=https%3A%2F%2Ftubaf.qucosa.de%2Fapi%2Fqucosa%253A31413%2Fmets)
-   [A Collaborative Effort to Improve Lossy Compression Methods for Climate Data](https://www.researchgate.net/profile/Peter_Lindstrom/publication/337322546_A_Collaborative_Effort_to_Improve_Lossy_Compression_Methods_for_Climate_Data/links/5dd1fcb24585156b351bc21c/A-Collaborative-Effort-to-Improve-Lossy-Compression-Methods-for-Climate-Data.pdf)
-   [IDEALEM: Statistical Similarity BasedData Reduction](https://arxiv.org/pdf/1911.06980.pdf)
-   [An Adaptive Checkpoint Model For Large-Scale HPC Systems](https://sc19.supercomputing.org/proceedings/tech_poster/poster_files/rpost265s2-file3.pdf)
-   [Analyzing the Performance and Accuracy of LossyCheckpointing on Sub-iteration of NWChem](https://sc19.supercomputing.org/proceedings/workshops/workshop_files/ws_drbsd106s1-file1.pdf)
-   [Towards Improving Rate-Distortion Performance of Transform-Based Lossy Compression for HPC Datasets](https://ieeexplore.ieee.org/abstract/document/8916286)
-   [Exploring Lossy Compression of Gene Expression Matrices](https://sc19.supercomputing.org/proceedings/workshops/workshop_files/ws_drbsd107s1-file1.pdf)
-   [A High Efficiency Multistage Coder for Lossless Audio Compression using OLS+ and CDCCR method](https://scholar.google.de/scholar_url?url=https://www.mdpi.com/2076-3417/9/23/5218/pdf&hl=en&sa=X&d=14054172226726273300&scisig=AAGBfm2V7_shUrpqsf7ZvAhqZmLkKOTRew&nossl=1&oi=scholaralrt)
-   [Evaluating Array DBMS Compression Techniques for Big Environmental Datasets](https://ieeexplore.ieee.org/abstract/document/8924326)
-   [Understanding GPU-Based Lossy Compression for Extreme-Scale Cosmological Simulations](http://tao.cs.ua.edu/paper/IPDPS20-Foresight.pdf)
-   [waveSZ: A Hardware-Algorithm Co-Design of Efficient Lossy Compression for Scientific Data](http://tao.cs.ua.edu/paper/PPoPP20-waveSZ.pdf)
-   [FRaZ: A Generic High-Fidelity Fixed-Ratio Lossy Compression Framework for Scientific Floating-point Data](https://arxiv.org/pdf/2001.06139.pdf)
-   [QFib: Fast and Efficient Brain Tractogram Compression](https://hal.telecom-paris.fr/hal-02454772/document)
-   [Toward Feature-Preserving 2D and 3D Vector Field Compression](https://www.mcs.anl.gov/~hguo/publications/LiangGDCRLOCP20-small.pdf)
-   [Parallelization of Variable Rate Decompression through Metadata](https://www.researchgate.net/publication/340101963_Parallelization_of_Variable_Rate_Decompression_through_Metadata?pli=1&loginT=OBtmoRJvSQilEPYH51H5FixTzbGpl0uwnMrPOVCZ-E7G7j0G_oX0UN8aV7lwnpz4fTm5sgqtuPKmlF3U&uid=5fhYri6ectA9kYVFMawpQHgyHyYbP1zp1rCu&cp=re442_pb_hnsg_sgp_naas_p410&ch=reg&utm_medium=email&utm_source=researchgate&utm_campaign=re442&utm_term=re442_pb_hnsg_sgp_naas&utm_content=re442_pb_hnsg_sgp_naas_p410)
-   [Seismic-ZFP: Fast and Efficient Compression and Decompression of Seismic Data](https://www.earthdoc.org/content/papers/10.3997/2214-4609.202032080)
-   [Exploring variable accuracy storage through lossy compression techniques](https://hal.inria.fr/hal-02572910/document)
-   [IDEALEM: Statistical Similarity Based Data Reduction](https://arxiv.org/abs/1911.06980)
-   [High-Throughput Convolutional Neural Network on an FPGA by Customized JPEG Compression](https://ieeexplore.ieee.org/abstract/document/9114817)
-   [HCompress: Hierarchical Data Compression for Multi-Tiered Storage Environments](http://www.cs.iit.edu/~scs/assets/files/hcompress.pdf)
-   [CUSZ: An Efficient GPU-Based Error-Bounded Lossy Compression Framework for Scientific Data](https://arxiv.org/pdf/2007.09625.pdf)
-   [Significantly Improving Lossy Compression for Scientific Data Sets Based on Multidimensional Prediction and Error-Controlled Quantization](https://ieeexplore.ieee.org/stamp/stamp.jsp?arnumber=7967203)
-   [Significantly Improving Lossy Compression for HPC Datasets with Second-Order Prediction and Parameter Optimization](https://dl.acm.org/doi/pdf/10.1145/3369583.3392688)
-   [Significantly Improving Lossy Compression Quality Based on anOptimized Hybrid Prediction Model](https://dl.acm.org/doi/pdf/10.1145/3295500.3356193)
-   [AMM: Adaptive Multilinear Meshes](https://www.researchgate.net/publication/343333710_AMM_Adaptive_Multilinear_Meshes?pli=1&loginT=EiFsHt1bq4GVyqB_KlqC7OauVatCQpnAE9QIf82OOlMmsRzF6W5MM63swxUpp4-RBZU3hypoLwkF3da7&uid=c6KMgX6xnJ5CKNBfR03pumKqebNM3xbp0ifR&cp=re442_pb_hnsg_sgp_naas_p410&ch=reg&utm_medium=email&utm_source=researchgate&utm_campaign=re442&utm_term=re442_pb_hnsg_sgp_naas&utm_content=re442_pb_hnsg_sgp_naas_p410)

<a id="org74ecb8c"></a>

# Predictive model training

-   [Development of Supervised Learning Predictive Models for Highly Non-linear Biological, Biomedical, and General Datasets](https://www.researchgate.net/publication/339228675_Development_of_Supervised_Learning_Predictive_Models_for_Highly_Non-linear_Biological_Biomedical_and_General_Datasets)

<a id="org5c4a25e"></a>

# Variable Precision

-   [Variable Precision Computing](https://www.osti.gov/servlets/purl/1573151)

<a id="orgf99d299"></a>

# General

-   [In-Situ Processing in Climate Science](https://link.springer.com/chapter/10.1007/978-3-030-34356-9_46)
-   [Entropy-Aware I/O Pipelining for Large-Scale Deep Learning on HPC Systems](https://ieeexplore.ieee.org/document/8526881)
